src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const getAllPersons = async () => {
+    setIsLoading(true);
+
+    try {
+      const allPersons = await getPeople();
+
+      setPeople(allPersons);
+    } catch {
+      setErrorMessage('Something went wrong');
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    getAllPersons();
+  }, []);
```

Comment:
I would rather inline this function, to let the linter track the deps of the effect hook

```suggestion
  

  useEffect(() => {
    const getAllPersons = async () => {
      setIsLoading(true);

      try {
        const allPersons = await getPeople();

        setPeople(allPersons);
      } catch {
        setErrorMessage('Something went wrong');
      } finally {
        setIsLoading(false);
      }
    };
  
    getAllPersons();
  }, []);
```